### PR TITLE
Duplicate move

### PR DESCRIPTION
**Pull request by Alotor**

### CODE REVIEW

File: frontend/src/uxbox/main/data/workspace/transforms.cljs [hunk 1]
```diff
@@ -206,33 +208,53 @@
...
-  (ptk/reify ::start-move
-    ptk/UpdateEvent
-    (update [_ state]
-      (-> state
-          (assoc-in [:workspace-local :transform] :move)))
-
+             (rx/with-latest vector ms/mouse-position-alt)
+             (rx/flat-map
+              (fn [[_ alt?]]
+                (prn "ALT" alt?)
```
Comment (niwinz) [inline frontend/src/uxbox/main/data/workspace/transforms.cljs]: :fire:

File: frontend/src/uxbox/main/ui/utils.cljs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+;; defined by the Mozilla Public License, v. 2.0.
+;;
+;; Copyright (c) 2020 UXBOX Labs SL
+
+(ns uxbox.main.ui.utils
+  (:require
+   [rumext.alpha :as mf]
+   [beicon.core :as rx]))
+
+(defn use-stream
```
Comment (niwinz) [inline frontend/src/uxbox/main/ui/utils.cljs]: I think we can add this function to uxbox.main.ui.hooks ns that already has many of them.

File: frontend/src/uxbox/main/data/workspace/transforms.cljs (hunk 1 above)
Comment (Alotor) [reply frontend/src/uxbox/main/data/workspace/transforms.cljs]: Done

File: frontend/src/uxbox/main/ui/utils.cljs (hunk 2 above)
Comment (Alotor) [reply frontend/src/uxbox/main/ui/utils.cljs]: You're right

**Pull request merged**